measurements/api/measurements.py

```diff
@@ -183,6 +208,25 @@ def list_measurements(
             probe_asn = probe_asn[2:]
         probe_asn = int(probe_asn)
 
+    # When the user specifies a list that includes all the possible values for
+    # boolean arguments, that is logically the same of applying no filtering at
+    # all.
+    if failure is not None:
+        if set(clean_list_arg(failure)) == set(['true', 'false']):
+            failure = None
+        else:
+            failure = failure == 'true'
+    if anomaly is not None:
+        if set(clean_list_arg(anomaly)) == set(['true', 'false']):
+            anomaly = None
+        else:
+            anomaly = anomaly == 'true'
+    if confirmed is not None:
+        if set(clean_list_arg(confirmed)) == set(['true', 'false']):
+            confirmed = None
+        else:
+            confirmed = confirmed == 'true'
+
```

Comment:
I don't get the logic. `probe_asn` is always SINGLE value, while `failure` is boolean and may be list of values. It looks like some inconsistency to me. What's the reason of it? IMHO, there should be some comment on that if that's intentional.

Reply:
I think you mean to say `failure` or `anomaly`. In any case the logic here is that I may want to look for `anomaly == True || anomaly == False`, which is different than asking for `anomaly == True`. That is encoded explicitly as `?anomaly=true,false`. Effectively that is equivalent to not imposing any filtering at all.

I can clarify this in the comment though.

Reply:
Okay, here is #35